src/components/Nav/UserFormComponent/userForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import UserFormInputs from './UserFormInputComponent/UserFormInputs';
+import UserFormBtns from './UserFormBtnComponent/UserFormBtns';
+
+import {
+  UserFormContainer,
+  UserFormBg,
+  UserFormBox,
+  UserFormLogo,
+  UserFormLogoImg,
+  UserFormText,
+  AskAccount,
+  CloseBtn,
+} from './userFormEle';
+
+function UserForm({ modalClose, form }) {
+  return (
+    <UserFormContainer>
+      <UserFormBg>
+        <UserFormBox paddingBig={form.paddingBig}>
+          <UserFormLogo>
+            <UserFormLogoImg src={Img} alt="UserFormImg" />
+          </UserFormLogo>
+          <UserFormText>{form.status}</UserFormText>
+          <UserFormInputs form={form} />
+          <UserFormBtns form={form} />
+          <AskAccount>{form.bottomText}</AskAccount>
+          <CloseBtn onClick={() => modalClose()}>X</CloseBtn>
```

Comment:
```suggestion
          <CloseBtn onClick={modalClose}>X</CloseBtn>
```
즉시 실행함수는 함수 인스턴스를 새로 만들기 때문에 만약 인자를 별도로 넘겨주는 상황이 아니라면 그냥 함수 자체를 넘겨주시면 될거 같습니다